platform/ios/PolyPodApp/PodApi/PostOffice.swift

```diff
@@ -391,13 +391,13 @@ extension PostOffice {
         PodApi.shared.polyNav.setTitle(title: title) { res, error in
         }
     }
-
+    
     private func handlePolyNavSetActiveAction(args: [Any]) {
         let actions = args[0] as! [String]
         PodApi.shared.polyNav.setActiveActions(actions: actions) { res, error in
         }
     }
-
+    
```

Comment:
These spaces possibly not needed, although I'm good with the additional CR.

Reply:
This is everywhere due to swift/xCode configs